config/motd.txt

```diff
@@ -1,5 +1,11 @@
 <!-- Most valid html will work in here, excluding images. -->
 
-<h1>Welcome to Space Station 13!</h1>
+<h1>Welcome to Fulpstation!</h1>
```

Comment:
Again, this is config, this should be done locally by wind.

Reply:
Once again, that was back when it wasn't done on the repo and I just wanted to help out Wind at that time. It doesn't really matter anymore, so I'll probably just revert that.